src/EnergyPlus/DataGlobals.hh

```diff
@@ -149,7 +149,6 @@ struct DataGlobal : BaseGlobalStruct
     bool AnyEnergyManagementSystemInModel = false; // true if there is any EMS or Erl in model.  otherwise false
     bool AnySurfPropOverridesInModel = false;      // true if there is any EMS or Erl overriding the surface properties for any surface.
     bool AnyConstrOverridesInModel = false;        // true if there is any EMS or Erl overriding the constructions for any surface.
-    bool anyPluginsOrCallbacksInModel = false;     // true if there is any plugin or active callback
```

Comment:
I like the idea of a global like this, as it was my idea to send @mjwitte down this road, but after seeing how the AnyEnergyManagementSystemInModel was already being used, I realized the new one wasn't needed right now.